src/platforms/eglstream-kms/server/kms_display_configuration.cpp

```diff
@@ -72,6 +75,26 @@ kms_connector_type_to_output_type(uint32_t connector_type)
     return static_cast<mg::DisplayConfigurationOutputType>(connector_type);
 }
 
+void name_outputs(std::vector<std::shared_ptr<mge::kms::EGLOutput>>& outputs)
+{
+    std::map<mg::DisplayConfigurationCardId, std::map<mg::DisplayConfigurationOutputType, int>> card_map;
+
+    for (auto& output : outputs)
+    {
+        auto const type = output->type;
+        auto const index_by_type = ++card_map[output->card_id][type];
```

Comment:
This might be because I'm still sick, but I had to stare at this a bit to work out what it's actually doing.

But what it's doing is sensible.